Modules/ThirdParty/HDF5/src/itkhdf5/c++/src/h5c++.in

```diff
@@ -172,7 +172,7 @@ fi
 case "$CXX" in
   gcc)
     kind="gcc"
-    ;;
+    ;
```

Comment:
This is a shell script. As such, double semicolon has a special meaning, see e.g.  https://stackoverflow.com/questions/16905183/dash-double-semicolon-syntax

So are most of the other files in this change list.